content/locales/TRANSLATIONS.md

```diff
@@ -0,0 +1,357 @@
+    "Subscribe": "Suscríbete",
+    "Subscribe to": "Suscríbete a",
+    "youremail@example.com": "tucorreo@ejemplo.com",
+    "1 post": "1 artículo",
+    "% posts": "% artículos"
+}
+```
+
+As you can see, it's usually just plain English on the left, and the language you choose on the right. Any code is optional and not required.
+
+For example, optionally, any translation key can be used on the left and on the templates, but readable English is advisable for simplicity and to take advantage of fallback to the text inside the `{{t}}` translation helper when no translation is available.
+
+Dates, with month names, are automatically translated. You don't need to include them in the translation files. 
+
+For a translatable clone of the Casper 1.4 theme (versions 1.4 and 2.x have different designs), see [WorldCasper1 &#x27B6;](https://github.com/juan-g/WorldCasper1/tree/i18n-translatable-frontend).
+
+### 4. Reactivate the translation file
+
+After editing a theme translation file, if it was the active file, to make the changes effective go to the admin panel, activate another language/locale -or another theme-, then switch back. In this way the translation file will be reloaded.
+
+### 5. Share the translation file with the theme maintainers
+
+If you send your language file to the maintainers, others can benefit and also help to keep the file updated for next theme versions.
+
+## How to Make Any Theme Translatable
+
+Ghost's themes use a very simple translation system, just for themes. On the other hand, server side messages for admins have a different system (although back-end translations are not yet enabled for the current versions) and will continue to use code in *.js* files with translation keys, such as `i18n.t('warnings.helpers.pagination.nextPrevValuesMustBeNumeric')`. However, to design and customize themes, just plain English is more user friendly, even when any translation keys can also optionally be used.
+
+For example, in *.hbs* theme templates:
+
+```
+{{t "Get the latest posts delivered right to your inbox"}}
+```
+
+### 1. Copy three core templates into the theme
+
+To translate front-end core *.hbs* templates, shared by all themes, three core templates should be overridden by copies included within the theme:
+
+- In the theme folder *content/themes/mytheme*:
+  - *subscribe.hbs* - copy of *core/server/apps/subscribers/lib/views/subscribe.hbs*
+
+- In the theme folder *content/themes/mytheme/partials*:
+  - *pagination.hbs* - copy of *core/server/helpers/tpl/pagination.hbs*
+  - *subscribe_form.hbs* - copy of *core/server/helpers/tpl/subscribe_form.hbs*
+
+### 2. Look for text on the theme's .hbs templates
+
+They contain the English text to translate, that is the text visible to site visitors. This also includes the copies overriding three core front-end templates.
+
+### 3. Wrap it in {{t}} translation helpers
+
+Just copy and paste `{{t "` to the left, and `"}}` to the rigth.
```

Comment:
Typo! `rigth` should be `right` 😄

Reply:
Oops... You're ~~rigth~~ right, I'll fix it. :wink: